.github/workflows/test-action.yml

```diff
@@ -14,9 +14,8 @@ jobs:
         #   for a uses:
         uses: sanketnaik99/changelog-on-pr@main
         with:
-          pull_number: 12
           token: ${{ secrets.GITHUB_TOKEN }}
-          labels: fixes,breaking
-          titles: FIXES,BREAKING
+          labels: breaking,fixes
+          titles: Fixes 🔧,Breaking Changes
```

Comment:
huh why are these in reverse order? does it matter? also are we adding another distinction now for breaking changes? i mean technically all of lucas' PRs will break things :pepe-giggles:

Reply:
@cellozeke This was a test PR lol.